lib/jnpr/jsnapy/jsnapy.py

```diff
@@ -373,8 +373,10 @@ def generate_rpc_reply(self, dev, output_file, hostname, config_data):
                     tfile, extra=self.log_detail)
 
         g = Parser()
+        command_list = []
```

Comment:
Add a comment here to make it explicit that command_list and rpc_list persist across test files and as they are passed by reference to generate_reply they help achieve command and rpc uniqueness